internal/icinga2/api_responses.go

```diff
@@ -103,6 +112,19 @@ type Downtime struct {
 	IsFixed bool `json:"fixed"`
 }
 
+// MarshalLogObject implements the zapcore.ObjectMarshaler interface.
+func (d *Downtime) MarshalLogObject(encoder zapcore.ObjectEncoder) error {
+	encoder.AddString("host", d.Host)
+	if d.Service != "" {
+		encoder.AddString("service", d.Service)
+	}
+
+	encoder.AddString("author", d.Author)
+	encoder.AddBool("fixed", d.IsFixed)
+	encoder.AddString("comment", d.Comment)
```

Comment:
Maybe keep this in the same order as the struct fields? Also, why did you exclude `RemoveTime`?

Reply:
> > Maybe keep this in the same order as the struct fields?
> 
> I actually did this on purpose, as you can have huge downtime comments and the more important with far too short and relevant values should come first IMHO.

Makes sense. It just felt a bit.. "untidy" for me :upside_down_face:

> > Also, why did you exclude RemoveTime?
> 
> I just didn't want to add too much information that isn't needed for debugging in anyway, or do you have use cases where this might be relevant? The `was_cancelled` key should be added though.

I also see your point here. No, at the moment I don't have a need for this field, but this might be in the future and then one either misses the information due to its unavailability or has to patch this again.